mlflow/langchain/_langchain_autolog.py

```diff
@@ -393,8 +387,10 @@ def _log_optional_artifacts(autolog_config, run_id, result, self, func_name, *ar
                     )
             except Exception as e:
                 _logger.warning(f"Failed to log model due to error {e}.")
-            if _update_langchain_model_config(self):
-                self.model_logged = True
+        # only try logging model once, even if it can't be logged
+        # we don't want to spam the user with warnings/infos
+        if _update_langchain_model_config(self):
+            self.model_logged = True
```

Comment:
```suggestion
            self._mlflow_model_logged = True
```

can we add `_mlflow` to the attributes set by mlflow for safety?